uv.lock

```diff
@@ -45,8 +45,8 @@ name = "astunparse"
 version = "1.6.3"
 source = { registry = "https://pypi.org/simple" }
 dependencies = [
-    { name = "six" },
-    { name = "wheel" },
+    { name = "six", marker = "python_full_version < '3.13' and sys_platform != 'darwin'" },
+    { name = "wheel", marker = "python_full_version < '3.13' and sys_platform != 'darwin'" },
```

Comment:
Do you know why `uv lock` is adding these markers? Is there any package incompatible with 3.13 and darwin?

Reply:
No idea. They are gone now after updating the lock file.